src/tests/test_meta_blk_mgr.cpp

```diff
@@ -776,6 +781,31 @@ TEST_F(VMetaBlkMgrTest, random_dependency_test) {
     this->shutdown();
 }
 
+TEST_F(VMetaBlkMgrTest, recovery_test) {
```

Comment:
Can we add this test to conan build test by default, e.g. add this test to the googlt test filter?

Reply:
yes, now I had add this unit tests to set of default test cases of HS metablk

Reply:
is this not committed to this PR yet? Sorry, I didn't find where this test case was made default.